.github/workflows/prod-release.yaml

```diff
@@ -12,136 +12,92 @@ permissions:
 jobs:
   prod-release:
     name: prod-release
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-22.04
     permissions:
       id-token: write
       contents: read
     steps:
-      - name: Configure AWS Credentials (beta)
-        uses: aws-actions/configure-aws-credentials@v1
+      - name: Checkout Code
+        uses: actions/checkout@v3
+
+      - name: Deploy Images To ECR Public
+        uses: ./.github/actions/push-image
         with:
-          aws-region: us-west-2
-          role-to-assume: ${{ secrets.BETA_AWS_ROLE }}
-          role-session-name: ControllerProdRelease
-      - name: Pull docker images
-        run: |
-          aws ecr get-login-password --region us-west-2 | docker login --username AWS --password-stdin ${{ secrets.BETA_AWS_ACCOUNT }}.dkr.ecr.us-west-2.amazonaws.com
-          docker pull ${{ secrets.BETA_AWS_ACCOUNT }}.dkr.ecr.us-west-2.amazonaws.com/amazon/appmesh-controller:${{ github.event.inputs.tag }}
-          docker pull ${{ secrets.BETA_AWS_ACCOUNT }}.dkr.ecr.us-west-2.amazonaws.com/amazon/appmesh-controller:${{ github.event.inputs.tag }}-linux_arm64
-      - name: Configure AWS Credentials (prod)
-        uses: aws-actions/configure-aws-credentials@v1
+          src_host: "${{ secrets.BETA_AWS_ACCOUNT }}.dkr.ecr.us-west-2.amazonaws.com"
```

Comment:
You changed the beta-release workflow to use secrets.CI_AWS_ACCOUNT but here you're still using secrets.BETA_AWS_ACCOUNT. I could use some context to understand what's going on between these.

Reply:
Standardized on BETA - these are the same values now; this patch just had it's roots back when CI was a distinct value that was used for beta image building and testing.